packages/core/src/components/tag/_common.scss

```diff
@@ -27,6 +27,10 @@ $tag-padding-large: ($tag-height-large - $pt-icon-size-large) / 2 !default;
   color: $pt-dark-text-color;
   font-size: $pt-font-size-small;
 
+  &.#{$ns}-active {
```

Comment:
ah dang actually i think this should be `&:focus` here, just to adjust the outline-offset.

and bring back this selector in `.pt-tag-input .pt-tag` like you had earlier

😄

Reply:
pushed another commit, hopefully I got it right this time!